master/job.go

```diff
@@ -0,0 +1,164 @@
+package master
+
+import (
+	"bytes"
+
+	"github.com/aaronang/cong-the-ripper/lib"
+)
+
+// NOTE: consider using math/big for some of the operations in this package
+
+var charSetSlice [][]byte
+
+func init() {
+	nums := "0123456789"
```

Comment:
Shouldn't all these _character sets_ be `const`? They shouldn't be allowed to change right?